test/integration_test.sh

```diff
@@ -244,6 +244,13 @@ python $script_dir/create_lira_config.py \
     --pipeline_tools_prefix $pipeline_tools_prefix > config.json
 
 # 7. Start Lira
+
+lira_container_name=lira
+
+# Check if an old container exists
```

Comment:
Maybe add an echo "checking for old container" or something here so there is more context for the following echo lines